src/components/BookingPanel/BookingPanel.js

```diff
@@ -10,10 +10,12 @@ import { formatMoney } from '../../util/currency';
 import { parse, stringify } from '../../util/urlHelpers';
 import config from '../../config';
 import { ModalInMobile, Button } from '../../components';
-import { BookingDatesForm } from '../../forms';
+import { BookingDatesForm, BookingDatesTimesForm } from '../../forms';
 
 import css from './BookingPanel.module.css';
-
+const LISTING_TYPE = {
+  equipment: 'equipment',
```

Comment:
This Listing_Type const was Duplicated in many files, why don't we declare it in only 1 constant file and export to other files?